docs/cobbler-conf/settings-yaml.rst

```diff
@@ -206,21 +209,51 @@ default: Depending on your distro. See values below.
 grub2_mod_dir
 #############
 
-TODO
+This is the folder where Cobbler will look for GRUB modules that are required for "cobbler mkloaders".
+
+default: Depending on your distro. See values below.
```

Comment:
I have noticed a lot of `directory` vs `folder` in the Cobbler docs. IMHO using "folder" is a bit weird in a Linux-based system.

Also, you can typically use present tense instead of future tense, which is then shorter and easier to understand for readers.

```suggestion
The directory where Cobbler looks for GRUB modules that are required for "cobbler mkloaders".

default: Depends on your distribution. See values below.
```